modules/gitops-creating-user-defined-cluster-roles-and-configuring-user-defined-permissions-for-application-controller.adoc

```diff
@@ -0,0 +1,201 @@
++
+[TIP]
+====
+Alternatively, you can use the web console to create a user-defined cluster role from the *Administrator* perspective. You can go to *User Management* -> *Roles* -> *Create Role*, use the preceding YAML template to add permissions, and click *Create*.
+====
++
+.Example output
+[source,terminal]
+----
+clusterrole.rbac.authorization.k8s.io/user-application-controller created
+----
++
+A user-defined cluster role is created. 
+
+. Verify that the `<argocd_name>-<argocd_namespace>-argocd-application-controller-admin` cluster role inherits permissions from the user-defined cluster role, by running the following command:
++
+[source,terminal]
+----
+$ oc get ClusterRole/<argocd_name>-<argocd_namespace>-argocd-application-controller-admin -o yaml
+----
++
+.Example output
+[source,terminal]
+----
+aggregationRule:
+  clusterRoleSelectors:
+  - matchLabels:
+      app.kubernetes.io/managed-by: spring-petclinic
+      argocd/aggregate-to-admin: "true"
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  annotations:
+    argocds.argoproj.io/name: example
+    argocds.argoproj.io/namespace: spring-petclinic
+    kubectl.kubernetes.io/last-applied-configuration: |
+      {"apiVersion":"argoproj.io/v1beta1","kind":"ArgoCD","metadata":{"annotations":{},"name":"example","namespace":"spring-petclinic"},"spec":{"aggregatedClusterRoles":true}}
+  creationTimestamp: "2024-08-14T09:59:15Z"
+  labels:
+    app.kubernetes.io/managed-by: spring-petclinic
+    app.kubernetes.io/name: example
+    app.kubernetes.io/part-of: argocd
+    argocd/aggregate-to-controller: "true"
+  name: example-spring-petclinic-argocd-application-controller-admin
+  resourceVersion: "79202"
+  uid: e2d35b6f-0832-4993-8b24-915a725454f9
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - namespaces
+  - persistentvolumeclaims
+  - persistentvolumes
+  - configmaps
+  verbs:
+  - '*'
+- apiGroups:
+  - compliance.openshift.io
+  resources:
+  - scansettingbindings
+  verbs:
+  - '*'
+----
++
+[TIP]
+====
+Alternatively, you can use the {OCP} web console to verify from the *Administrator* perspective. You can go to *User Management* -> *Roles*, use the *Filter* option, select *Cluster-wide Roles*, and search for the `<argocd_name>-<argocd_namespace>-argocd-application-controller-admin` cluster role. You have to open the cluster role to check the details and configurations.
+====
+
+. Verify that the `<argocd_name>-<argocd_namespace>-argocd-application-controller` aggregated cluster role inherits permissions from the `<argocd_name>-<argocd_namespace>-argocd-application-controller-admin` and `<argocd_name>-<argocd_namespace>-argocd-application-controller-view` cluster roles, by running the following command:
++
+[source,terminal]
+----
+$ oc get ClusterRole/<argocd_name>-<argocd_namespace>-argocd-application-controller -o yaml
```

Comment:
need a "where" sequence for the user-replaceable values

Reply:
Addressed